include/exadg/aero_acoustic/single_field_solvers/analytical_time_int_fluid.h

```diff
@@ -0,0 +1,198 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2024 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef INCLUDE_EXADG_AERO_ACOUSTIC_SINGLE_FIELD_SOLVERS_ANALYTICAL_TIME_INT_FLUID_H_
+#define INCLUDE_EXADG_AERO_ACOUSTIC_SINGLE_FIELD_SOLVERS_ANALYTICAL_TIME_INT_FLUID_H_
+
+#include <exadg/incompressible_navier_stokes/time_integration/time_int_bdf.h>
+
+namespace ExaDG
+{
+namespace IncNS
+{
+/**
+ * A class that sets the analytical solution for the next time instead of computing it.
+ * To be able to compute temporal derivatives etc. this class is based on TimeIntBDF.
+ */
+template<int dim, typename Number>
+class TimeIntAnalytic : public TimeIntBDF<dim, Number>
```

Comment:
Did you think about placing this class/code in a more central place where the other fluid time integrators live? What is your opinion on this?

I am not completely happy with the name `TimeIntAnalytic` (since "analytic time integrator" might be understood differently than what is done here, but together with the class description I am principally also fine with the current name. I am thinking of `TimeIntInterpolateAnalyticalSolution` or `TimeIntInjectAnalyticalSolution`. Maybe this is clearer; unfortunately it is also much longer.

Reply:
[954d775](https://github.com/exadg/exadg/pull/659/commits/954d775f6c6149875632294d9d98b68d9604e32f)